celestia/peers.txt

```diff
@@ -1 +1 @@
-
+45779d388b18ad48d98c6ad54f35e95fe2a3b35a@165.22.30.175:26656
```

Comment:
Can you remove this change please? `celestia/DSRV.txt` is sufficient.

Reply:
Do I remove the file or add an empty `peers.txt` file?